login.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require_once 'connectToDB.php';
+require_once 'src/Models/UsersModel.php';
+
+session_start();
+function verifyLogin(): void
+{
+    if (isset($_POST['username'])) {
+
+        $inputtedUsername = $_POST['username'];
+        $inputtedPassword = $_POST['password'];
+
+        $hashedPassword = password_hash($inputtedPassword, PASSWORD_BCRYPT);
+
+        $db = connectToDB();
+        $usersModel = new UsersModel($db);
+
+        $users = $usersModel->selectUser($inputtedUsername);
+        if ($users === null) {
+            echo 'User does not exist';
+        } else {
+            $storedPassword = $users->password;
+            $storedUsername = $users->username;
+
+            if ((password_verify($storedPassword, $hashedPassword)) && ($inputtedUsername === $storedUsername)) {
+                $_SESSION['userid'] = $users->id;
+                $_SESSION['username'] = $users->username;
```

Comment:
There will be a few different things you need to do with the session for logging users in

Suggest you make a new class to handle login session stuff for you. It could have methods like:

login user - does the lines highlighted here
check if a user is logged in - useful for the redirect comment I posted in this file, and also when you're hiding bits of the page to logged in users